### PR TITLE
dev/rest: Update pause/resume to syncthing/syncthing#3989

### PR DESCRIPTION
**Pull request by imsodin**
syncthing/syncthing#3989

### CODE REVIEW

**Comment by ProactiveServices** [comment]
@st-review lgtm

**Comment by st-review** [comment]
@ProactiveServices: Noted! Need another LGTM or explicit merge command.

File: rest/system-pause-post.rst [hunk 1]
```diff
@@ -1,6 +1,8 @@
 POST /rest/system/pause
 =======================
 
-Pause the given device.
+Pause the given device or all devices.
 
-Takes the mandatory ``device`` parameter and returns status 200 and no content upon success, or status 500 and a plain text error on failure.
+Takes the optional parameter ``device`` (device name). When ommitted,
```
Comment (imsodin) [inline rest/system-pause-post.rst]: It is actually the device name (as in UI, definitely better than device string), not the device ID. See:
https://github.com/syncthing/syncthing/blob/master/cmd/syncthing/gui.go#L1130

**Comment by calmh** [comment]
All REST methods that deal with devices take device *ID*s, nothing else.

**Comment by imsodin** [comment]
For some reason my mind made the connection string -> name and was thus confused by `DeviceIDFromString`. I should have looked what the function actually does. Thanks for the correction.

**Comment by calmh** [comment]
@st-review merge

**Comment by st-review** [comment]
:ok_hand: Merged as e58c571a60d3e958f442915783e2e7892b97326b. Thanks, @imsodin!

**Pull request closed without merging**